iceoryx_utils/cmake/IceoryxPlatform.cmake

```diff
@@ -48,3 +48,32 @@ if(BUILD_STRICT)
         set(ICEORYX_WARNINGS ${ICEORYX_WARNINGS} -Werror)
     endif (  )
 endif(BUILD_STRICT)
+
+if(sanitize)
+    if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang" OR CMAKE_CXX_COMPILER_ID STREQUAL "AppleClang")
+        # NOTE : This works only when iceoryx is built standalone , in which case CMAKE_SOURCE_DIR point to iceoryx_meta
+        set(ICEORYX_SANITIZER_BLACKLIST -fsanitize-blacklist=${CMAKE_SOURCE_DIR}/sanitizer_blacklist/asan_compile_time.txt)
```

Comment:
why not just putting the blacklist right to this file, then it works all the time?

Reply:
there might be a `beside` missing right of `right`

Reply:
> why not just putting the blacklist right to this file, then it works all the time?

This is the method [suggested](https://clang.llvm.org/docs/AddressSanitizer.html#suppressing-errors-in-recompiled-code-blacklist) 



> there might be a `beside` missing right of `right`

I dint quite get that !

Reply:
okay ..now I understand what you meant.
I am not sure how do I get the path to `IceoryxPlatform.cmake` (when this file is included in another cmake file)

Reply:
another possibility would be to use `file({WRITE | APPEND} <filename> <content>...)` from cmake and create those files on the fly in e.g. `CMAKE_BUILD_DIR`. It's up to you if you want to do the hassle